parsely/src/main/java/com/parsely/parselyandroid/ParselyCoroutineScope.kt

```diff
@@ -0,0 +1,7 @@
+package com.parsely.parselyandroid
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+
+val sdkScope = CoroutineScope(SupervisorJob() + Dispatchers.IO)
```

Comment:
Question (❓): Do we actually need the `SupervisorJob` here, or is it just a nice to have? 🤔

Reply:
Children of the `SupervisorJob` can fail/be canceled independently. This `CoroutineScope` is aimed for different jobs in the SDK, e.g. `FlushTimer` shouldn't fail if HTTP request did.

Reply:
> Children of the SupervisorJob can fail/be canceled independently. 

💯 

> This CoroutineScope is aimed for different jobs in the SDK

Oh, didn't know about this, thanks for clarifying that for me @wzieba ! 👍